src/python/tests/appengine/handlers/cron/oss_fuzz_build_status_test.py

```diff
@@ -396,7 +397,6 @@ def test_recovered_build_failure(self):
     self.assertEqual(0, data_types.OssFuzzBuildFailure.query().count())
 
     issue = self.itm.issues[1]
-    self.assertFalse(issue.open)
```

Comment:
we don't set it anymore. checking the status should be enough, because setting 'open' doesn't actually do anything when the API request is made to monorail. The status determines whether an issue is open or not.